config/config.go

```diff
@@ -0,0 +1,171 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/googleinterns/aso_sxs_viewer/proto"
+	"google.golang.org/protobuf/encoding/prototext"
+)
+
+type ViewerConfig struct {
+	*proto.ViewerConfig
+}
+
+type BrowserConfig struct {
+	Selector string
+	Position int
+	URL      string
+}
+
+// GetConfig generates .aso_sxs_viewer directory and config file, if it doesn't already exists.
+// It validates the pre-existing config if found.
+// You should avoid repeated calls to avoid validation overhead.
+func GetConfig() (ViewerConfig, error) {
```

Comment:
return a pointer to ViewerConfig everywhere (and make it hold proto.ViewerConfig (non pointer)) if you want. Returning nil, err looks more obvious than ViewerConfig{nil}, err